projects/Scion/default.nix

```
@@ -0,0 +1,20 @@
+{
+  pkgs,
+  lib,
+  sources,
+} @ args: {
+  packages = {
+    inherit
+      (pkgs)
+      scion
+      scion-apps
+      scion-bootstrapper
+      ioq3-scion
+      pan-bindings
+      ;
+  };
+  nixos = {
+    modules.services.scion = "${sources.inputs.nixpkgs}/nixos/modules/services/networking/scion/scion.nix";
+    tests.scion = "${sources.inputs.nixpkgs}/nixos/tests/scion/freestanding-deployment/default.nix";
```

```suggestion
    tests.scion = "${sources.inputs.nixpkgs}/nixos/tests/scion/freestanding-deployment/default.nix";
    examples = null;
```

There isn't any dedicated example, right? Need to skip `null` fields in the overview renderer.